src/main/java/org/sonarsource/sonarlint/ls/ServerMain.java

```diff
@@ -40,36 +45,80 @@ public static void main(String... args) {
     new ServerMain(System.out, System.err).startLanguageServer(args);
   }
 
+  static int getIndexOfParam(String param, String[] args) {
```

Comment:
Why not converting arguments array into an ArrayList, and using `ArrayList.indexOf` method?